Makefile

```diff
@@ -1,7 +0,0 @@
-build: build-7.10
-
-build-7.10:
-	STACK_YAML="stack-7.10.yaml" stack build
-
-build-8.0:
-	STACK_YAML="stack-8.0.yaml" stack build
```

Comment:
Why is this deleted? It should be updated or left alone.

Reply:
Why does it exist? It looks like cruft.

Reply:
@cydparser it's not cruft, it spares the user typing and makes them aware of the option to build that way. There is no way it saves people time net-net removing this versus leaving it in unless you're actually deleting and purging the Stack GHC 8.0 build.